### DISCUSSION TITLE
Standalone Domain-Cloaking Mode

### DISCUSSION THREAD

**Discussion by ghost** [DNSCrypt/dnscrypt-proxy#2590] [Ideas]
Windows native hosts file pales compared to DNSCrypt-Proxy domain cloaking, but DNSCrypt-Proxy doesn't have a bypass mode that allows users to continue using non-encrypted DNS (such as VPN DNS) while enjoying DNSCrypt-Proxy's domain cloaking feature. It would be nice to have such a bypass mode!

**Comment by lifenjoiner** [comment]
As it is named, it focuses on encrypted DNS.
You may want to take a look at https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Forwarding.

**Comment by ghost** [comment]
Yes, which is why I only mentioned this as an idea. DNSCrypt-Proxy has an incredible domain blocking/cloaking feature and caching mechanism, but 3rd party DNS servers (encrypted or not) will never provide as low of latency as VPN DNS servers do within corresponding VPN tunnels. 

Perhaps all that is necessary is a feature to forward all queries with some kind of a wildcard in forwarding rules, assuming query cloaking rules are processed before forward rules.
